Index: README.md
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision 0c98513887224bdc03d7a7ceff4c88d8faa29378)
+++ /dev/null	(revision 0c98513887224bdc03d7a7ceff4c88d8faa29378)
@@ -1,9 +0,0 @@
-BONJOUR LES KHEYYYYYZ ( Bonjour Monsieur)
-
-J'ai fais une 2ème branche pour vous pour le TD avec le docker l'api 
-
-La branche principale c'est pour le projet Web Data API. 
-
-On a décidé d'utiliser plusieurs langages en back end il y  aura du Python et du Java 
-
-Pour le Front on aura du HTML et du JS ( si on a le temps de le faire le front mais ça metonnerait ) 
diff --git a/API-Monstre-Invo/DockerfileInvocation b/API-Monstre-Invo/DockerfileInvocation
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/controller/MonsterController.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/controller/MonsterController.java
new file mode 100644
diff --git a/API-Monstre-Invo/docker-compose.yml b/API-Monstre-Invo/docker-compose.yml
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/MonsterRepository.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/MonsterRepository.java
new file mode 100644
diff --git a/API-Monstre-Invo/DockerfileMonstre b/API-Monstre-Invo/DockerfileMonstre
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Invocation.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Invocation.java
new file mode 100644
diff --git a/API-Monstre-Invo/.gitignore b/API-Monstre-Invo/.gitignore
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/resources/monstres.json b/API-Monstre-Invo/demo/src/main/resources/monstres.json
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/DemoApplication.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/DemoApplication.java
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/controller/InvocationController.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/controller/InvocationController.java
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/resources/application.properties b/API-Monstre-Invo/demo/src/main/resources/application.properties
new file mode 100644
diff --git a/API-Monstre-Invo/demo/mvnw b/API-Monstre-Invo/demo/mvnw
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/test/java/InvocationTest.java b/API-Monstre-Invo/demo/src/test/java/InvocationTest.java
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/classes/application.properties b/API-Monstre-Invo/demo/target/classes/application.properties
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/classes/monstres.json b/API-Monstre-Invo/demo/target/classes/monstres.json
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst b/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst b/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/createdFiles.lst b/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/createdFiles.lst
new file mode 100644
diff --git a/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst b/API-Monstre-Invo/demo/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst
new file mode 100644
diff --git a/API-Monstre-Invo/demo/HELP.md b/API-Monstre-Invo/demo/HELP.md
new file mode 100644
diff --git a/API-Monstre-Invo/demo/docker-dev-env/docker-compose.yml b/API-Monstre-Invo/demo/docker-dev-env/docker-compose.yml
new file mode 100644
diff --git a/API-Monstre-Invo/demo/mvnw.cmd b/API-Monstre-Invo/demo/mvnw.cmd
new file mode 100644
diff --git a/API-Monstre-Invo/demo/pom.xml b/API-Monstre-Invo/demo/pom.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/.gitignore b/API-Monstre-Invo/.idea/.gitignore
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/misc.xml b/API-Monstre-Invo/.idea/misc.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/vcs.xml b/API-Monstre-Invo/.idea/vcs.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/uiDesigner.xml b/API-Monstre-Invo/.idea/uiDesigner.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/encodings.xml b/API-Monstre-Invo/.idea/encodings.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/compiler.xml b/API-Monstre-Invo/.idea/compiler.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/modules.xml b/API-Monstre-Invo/.idea/modules.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/jarRepositories.xml b/API-Monstre-Invo/.idea/jarRepositories.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/mongoSettings.xml b/API-Monstre-Invo/.idea/mongoSettings.xml
new file mode 100644
diff --git a/API-Monstre-Invo/.idea/dbnavigator.xml b/API-Monstre-Invo/.idea/dbnavigator.xml
new file mode 100644
diff --git a/API-Monstre-Invo/target/maven-archiver/pom.properties b/API-Monstre-Invo/target/maven-archiver/pom.properties
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/DataLoader.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/DataLoader.java
new file mode 100644
diff --git a/API-Monstre-Invo/README.md b/API-Monstre-Invo/README.md
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/StatRatio.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/StatRatio.java
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Monster.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Monster.java
new file mode 100644
diff --git a/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Skill.java b/API-Monstre-Invo/demo/src/main/java/com/example/demo/beans/Skill.java
new file mode 100644
